tests/cases/elements/timer_start_stop_restart.slint

```diff
@@ -0,0 +1,215 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
+
+export component TestCase inherits Window {
+    out property <string> result1;
+    out property <string> result2;
+
+    // multiple timers so we can ensure that the index resolution is working correctly
```

Comment:
:+1: 

Could you add an extra timer in another component?

Like add some component such as

```slint
component ComponentWithTimer {
   timer-a := Timer { ... }
   if true: Rectangle { // Make sure it works in condition.
      timer-b := Timer { 
          running: false; 
          interval: 3s; 
          triggered => { self.stop(); root.triggered("b") }
          init => { timer-b.start() }
      } 
   } 
   
   callback trigerred(string);  // called from the triggered callback of the timers
   function ... // to start and restart timer-a
}
```

And then use that component twice, from the Window.

Just to make sure that nothing fishy happens when the timer is in a sub component.

This code is just an example feel free to be creative ;-)